src/components/NodeLink/NodeLink.vue

```diff
@@ -69,6 +73,18 @@ export default {
         edgeWidthKey: undefined,
       })
     },
+    renderNested: {
+      type: Boolean,
+      default: false
+    },
+    nestedBarVariables: {
+      type: Array[String],
+      default: () => []
```

Comment:
Why not just `[]` here?

Reply:
`Type of the default value for 'nestedGlyphVariables' prop must be a function` if it's not a function

Reply:
Still getting this with the new validator and type

Reply:
Huh I didn't realize this warning was coming from the linter. I guess this is to force a fresh copy of a mutable object (like a list) to be constructed to avoid weird problems with reference sharing. So follow the linter and keep the `default` value as is (but still use the type and validator we discussed before).